go/client/getentries.go

```diff
@@ -20,17 +21,26 @@ type getEntriesResponse struct {
 	Entries []leafEntry `json:"entries"` // the list of returned entries
 }
 
-// GetEntries attempts to retrieve the entries in the sequence [|start|, |end|] from the CT log server. (see section 4.6.)
-// Returns a slice of LeafInputs or a non-nil error.
-func (c *LogClient) GetEntries(start, end int64) ([]ct.LogEntry, error) {
+func rawGetEntries(ctx context.Context, httpClient *http.Client, logURL string, start, end int64) (*getEntriesResponse, error) {
```

Comment:
'rawGetEntries' is a slightly odd name.  getRawEntries,  getEntriesRaw?
'getEntriesResponse' would be nice but it's the name of a type so that'd be confusing.